cpp/include/proxy_protocol.h

```diff
@@ -151,6 +152,13 @@ void read_read_objects_slices2_response(message &m, Status &status,
                                         const std::vector<ObjectSlices> &dest,
                                         std::vector<object_info> &object_infos);
 
+void write_apply_sequence_request(
+    message_builder &mb, const string &namespace_, const bool write_barrier,
+    const std::vector<std::shared_ptr<alba::proxy_client::sequences::Assert>> asserts,
+    const std::vector<std::shared_ptr<alba::proxy_client::sequences::Update>> updates);
+
```

Comment:
Creates copies, should be `const std::vector<....>&` for both asserts and updates.